content/docs/command-reference/fetch.md

```diff
@@ -11,19 +11,29 @@ usage: dvc fetch [-h] [-q | -v] [-j <number>] [-r <name>] [-a] [-T]
                  [targets [targets ...]]
 
 positional arguments:
-  targets        Limit command scope to these stages or .dvc files.
-                 Using -R, directories to search for stages or .dvc
-                 files can also be given.
+  targets       Limit command scope to these tracked files/directories,
+                .dvc files, or stage names.
 ```
 
 ## Description
 
-The `dvc fetch` downloads DVC-tracked files from remote storage into the cache
-of the project, but without placing them in the <abbr>workspace</abbr>. This
-makes the data files available for linking (or copying) into the workspace.
-(Refer to [dvc config cache.type](/doc/command-reference/config#cache).) Along
-with `dvc checkout`, it's performed automatically by `dvc pull` when the target
-`dvc.yaml` or `.dvc` files are not already in the cache:
+`dvc fetch` downloads DVC-tracked files from remote storage into the cache of
+the project (without placing them in the <abbr>workspace</abbr>). This makes
+them available for linking (or copying) into the workspace (refer to
+[dvc config cache.type](/doc/command-reference/config#cache)).
+
+Without arguments, this ensures that all the files needed for all `.dvc` files
+and stages (in `dvc.yaml` and `dvc.lock`) in the workspace exist in the cache
```

Comment:
in this case, for example`, stages are defined in `dvc.yaml`,  no need to mention `dvc.lock`. Or mention it like this:

`and stages in `dvc.yaml` (as `dvc.lock` specifies)`